tests/ui/documents/test_subjects.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests subject classes for documents."""
+import mock
+import pytest
+from utils import mock_response
+
+from rero_ils.modules.contributions.api import Contribution
+from rero_ils.modules.documents.commons import SubjectFactory
+from rero_ils.modules.documents.models import DocumentSubjectType
+from rero_ils.modules.utils import get_ref_for_pid
+
+
+def test_document_local_subjects():
```

Comment:
This should be in the `unit` directory as this not requires a flask app. Moreover, it would be nice to add a test at the REST API to check the facets.